### ISSUE TITLE
BP activity comment area missing placeholder and aria-label

### ISSUE THREAD

**Issue by joost-abrahams** [Automattic/gutenberg-everywhere#166]
The placeholder and aria-label are missing

Placeholder = What's new, username?
Aria-label = Post what's new
